database_test.go

```diff
@@ -0,0 +1,38 @@
+package driver_test
+
+import (
+	"fmt"
+	"testing"
+
+	driver "github.com/arangodb/go-driver"
+	"github.com/stretchr/testify/assert"
```

Comment:
Please remove the `github.com/stretchr/testify` dependency.
We want this driver to be as self-containing as possible